### ISSUE TITLE
Not found: FeedInputs: unable to find feed output 

### ISSUE THREAD

**Issue by billtiger** [tensorflow/tensorflow#17586]
@GeorgeBohw
hello,i also meet this problem.could you tell me how do you solve this problem?
@tensorflowbutler 
i have use the the summarize_graph tool to guesses about likely input and output nodes,this is the command:bazel-bin/tensorflow/tools/graph_transforms/summarize_graph --in_graph=/home/biao/test/occ_detect.pb
here is the message:

No inputs spotted.
No variables spotted.
Found 1 possible outputs: (name=softmax_linear/softmax_linear_1, op=Add) 
Found 850090 (850.09k) const parameters, 0 (0) variable parameters, and 0 control_edges
Op types used: 18 Const, 13 Identity, 5 Add, 5 Relu, 3 BiasAdd, 3 MaxPool, 3 MatMul, 3 LRN, 3 Conv2D, 2 Mul, 1 Floor, 1 QueueDequeueManyV2, 1 RandomShuffleQueueV2, 1 RandomUniform, 1 RealDiv, 1 Reshape, 1 Sub
To use with tensorflow/tools/benchmark:benchmark_model try these arguments:
bazel run tensorflow/tools/benchmark:benchmark_model -- --graph=/home/biao/test/occ_detect.pb --show_flops --input_layer= --input_layer_type= --input_layer_shape= --output_layer=softmax_linear/softmax_linear_1

i guess the problem is my .pb file is fault?

**Comment by billtiger** [comment]
@

**Comment by tensorflowbutler** [comment]
Thank you for your post. We noticed you have not filled out the following field in the issue template. Could you update them if they are relevant in your case, or leave them as N/A? Thanks.
Have I written custom code
OS Platform and Distribution
TensorFlow installed from
TensorFlow version
Bazel version
CUDA/cuDNN version
GPU model and memory
Exact command to reproduce

**Comment by tensorflowbutler** [comment]
It has been 14 days with no activity and the `awaiting response` label was assigned. Is this still an issue?

**Comment by billtiger** [comment]
@tensorflowbutler 
I am sorry that I forgot to reply! I have solve this problem,the reason is my training codes has fault.
thank you remember me this issue.

**Comment by billtiger** [comment]
@cvJie 
my reason is that i forget put the input to the placeholder,you can check your training codes!
i think it also has other reasons if your training codes are ok!

**Comment by billtiger** [comment]
@cvJie 
my reason is that i forget put the input to the placeholder,you can check your training codes!
i think it also has other reasons if your training codes are ok!

**Comment by tensorflowbutler** [comment]
Nagging Assignee @shivaniag: It has been 15 days with no activity and this issue has an assignee. Please update the label and/or status accordingly.

**Comment by tensorflowbutler** [comment]
Nagging Assignee @shivaniag: It has been 30 days with no activity and this issue has an assignee. Please update the label and/or status accordingly.

**Comment by tensorflowbutler** [comment]
Nagging Assignee @shivaniag: It has been 45 days with no activity and this issue has an assignee. Please update the label and/or status accordingly.

**Comment by tensorflowbutler** [comment]
Nagging Assignee @shivaniag: It has been 60 days with no activity and this issue has an assignee. Please update the label and/or status accordingly.

**Comment by tensorflowbutler** [comment]
Nagging Assignee @shivaniag: It has been 75 days with no activity and this issue has an assignee. Please update the label and/or status accordingly.

**Comment by tensorflowbutler** [comment]
Nagging Assignee @shivaniag: It has been 90 days with no activity and this issue has an assignee. Please update the label and/or status accordingly.

**Comment by tensorflowbutler** [comment]
Nagging Assignee @shivaniag: It has been 105 days with no activity and this issue has an assignee. Please update the label and/or status accordingly.

**Comment by sachinprasadhs** [comment]
`Placeholder` is depreciated in Tensorflow 2.x version and Tensorflow 1.x is not supported anymore. Please try the latest [Tensorflow version](https://www.tensorflow.org/install/pip) and let us know if the problem still persists. Thanks!
